dvc/dependency/repo.py

```diff
@@ -114,6 +114,9 @@ def download(self, to):
                 self.def_path, self.def_repo[self.PARAM_URL]
             )
 
-    def update(self):
-        with self._make_repo(rev_lock=None) as repo:
+    def update(self, rev):
```

Comment:
Let's pass it by name for consistency.

Reply:
Hey, I'm new to Python and couldn't get what you mean. Can you please detail this a bit?

Reply:
@ilgooz I'm pretty sure he means:
```
def update(self, rev=None):
```
🙂

Reply:
done by 9d68c8aa